src/elements/content-sidebar/BoxAISidebarContent.tsx

```diff
@@ -101,35 +124,70 @@ function BoxAISidebarContent(props: ApiWrapperProps) {
         <>
             {renderBoxAISidebarTitle()}
             {isResetChatEnabled && <ClearConversationButton onClick={onClearAction} />}
+            <Tooltip content={formatMessage(messages.sidebarBoxAISwitchToModalView)} variant="standard">
+                <IconButton
+                    aria-label={formatMessage(messages.sidebarBoxAISwitchToModalView)}
+                    className="bcs-BoxAISidebar-expand"
+                    data-target-id="IconButton-expandBoxAISidebar"
+                    icon={ArrowsExpand}
+                    onClick={onSwitchToModalClick}
+                    size="small"
+                />
+            </Tooltip>
         </>
     );
 
     return (
         <AgentsProvider>
-            <SidebarContent
-                actions={renderActions()}
-                className="bcs-BoxAISidebar"
-                elementId={elementId}
-                sidebarView={SIDEBAR_VIEW_BOXAI}
-            >
-                <div className="bcs-BoxAISidebar-content">
-                    <BoxAiContentAnswers
-                        className="bcs-BoxAISidebar-contentAnswers"
-                        contentName={contentName}
-                        contentType={formatMessage(messages.sidebarBoxAIContent)}
-                        hostAppName={hostAppName}
-                        isAIStudioAgentSelectorEnabled={isAIStudioAgentSelectorEnabled}
-                        isStopResponseEnabled={isStopResponseEnabled}
-                        questions={questions}
-                        stopQuestion={stopQuestion}
-                        submitQuestion={sendQuestion}
-                        userInfo={userInfo}
-                        variant="sidebar"
-                        recordAction={recordAction}
-                        {...rest}
-                    />
-                </div>
-            </SidebarContent>
+            <>
```

Comment:
I don't think we need a `Fragment` here

Reply:
We do, `AgentsProvider` expects a single child of type Element.